sockets/docker/docker.go

```diff
@@ -153,11 +153,13 @@ func (p *Poller) getContainerInfo(container docker.APIContainers) (*sockets.Cont
 		return nil, err
 	}
 	environment := p.getEnvironment(inspected)
+	labels := inspected.Config.Labels
```

Comment:
Perhaps it's not really needed to introduce variables for `environment` and `labels`